PoC-differ/differ.go

```diff
@@ -390,11 +390,11 @@ func generateWeeklyNotificationMessage(advisorURI string, accountID string, dige
 	}
 
 	payload := types.EventPayload{
-		notificationPayloadTotalClusters:        string(digest.ClustersAffected),
-		notificationPayloadTotalRecommendations: string(digest.Recommendations),
-		notificationPayloadTotalIncidents:       string(digest.Incidents),
-		notificationPayloadTotalCritical:        string(digest.CriticalNotifications),
-		notificationPayloadTotalImportant:       string(digest.ImportantNotifications),
+		notificationPayloadTotalClusters:        fmt.Sprint(digest.ClustersAffected),
```

Comment:
dtto

Reply:
In this case yes. All the values in the digest struct are integers.

Reply:
true, sorry